source/indexes/compound-index.txt

```diff
@@ -0,0 +1,95 @@
+.. _kotlin-sync-compound-index:
+
+================
+Compound Indexes
+================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: index, query, optimization, efficiency
+
+Overview
+--------
+
+:manual:`Compound indexes </core/index-compound/>` hold references to multiple
+fields within a collection's documents, improving query and sort performance.
+
+When creating a compound index, you must specify the following:
+
+- The fields on which to create the index
+- The sort order for each field (ascending or descending)
```

Comment:
[q] Is there a default sort order if the sort order isn't specified or is specifying always required?

Reply:
It's required – [source](https://www.mongodb.com/docs/manual/core/indexes/index-types/index-compound/)